src/components/Table.tsx

```diff
@@ -9,34 +9,22 @@ import some = require('lodash/some');
 import sortBy = require('lodash/sortBy');
 import * as React from 'react';
 import FaSort = require('react-icons/lib/fa/sort');
+import { TableColumn, TableProps } from 'rendition';
 import styled from 'styled-components';
 
 import theme from '../theme';
 import Button from './Button';
 import { Flex } from './Grid';
 import Input from './Input';
 
-type TableSortFunction = <T>(a: T, b: T) => number;
+const highlightStyle = `
+	background-color: ${theme.colors.info.light};
 
-interface TableColumn<T> {
-	field: keyof T;
-	icon?: string;
-	label?: string | JSX.Element;
-	sortable?: boolean | TableSortFunction;
-	render?: (value: any, row: T) => string | JSX.Element;
-}
-
-export interface TableProps<T> {
-	columns: Array<TableColumn<T>>;
-	data: T[];
-	// Optionally provide a key that should be used as a unique identifier for each row
-	rowKey?: keyof T;
-
-	// Only usable if a rowKey property is also provided.
-	// If an onCheck property is provided , then checkboxes will be renders,
-	// allowing rows to be selected.
-	onCheck?: (checkedItems: T[]) => void;
-}
+	> td:first-child {
+		border-left: 3px solid ${theme.colors.info.main};
```

Comment:
Ultra-NIT: this could be just shadow so that the browser only repaints instead of rerendering to calculate the new width.

Reply:
Do you mean `box-shadow`?

Reply:
yea, sorry for not clarifying

Reply:
Ah yep, good idea